src/starbase/config.ts

```diff
@@ -20,6 +20,23 @@ const integrationSchema: Schema = {
   additionalProperties: true,
 };
 
+const storageSchema: Schema = {
+  type: 'object',
+  properties: {
+    engine: { type: 'string' },
+    config: { type: 'object' },
+  },
+  required: ['engine'],
+};
+
+const configSchema: Schema = {
+  type: 'object',
+  properties: {
+    integrations: { type: 'array' },
+    storage: { type: 'object' },
```

Comment:
```suggestion
    storage: storageSchema,
```

Reply:
Additionally, the `integrationSchema` `config` value should change to `configSchema`.